### PR TITLE
feat: initial setup

### PR DESCRIPTION
**Pull request by pfulton**
Adds `normalize.css` and CSS linting via Stylelint.

## Description

- Adds the `normalize` stylesheet to the project, and adds it to the site `<head>`.
- Adds CSS linting via Stylelint and adds an initial config for it.
  - Temporarily adds several of the other stylesheets to the `.stylelintignore` for now. We'll be replacing the styles that exist in those files eventually.
- Adds a `lint:css` task.
- Adds a `lint:js` task (renamed from the previous `lint` task)
- Updates the `lint` task to run both `lint:js` and `lint:css`.

## Related Issue
n/a

## Motivation and Context
Adding Stylelint should help with keeping our CSS syntax consistent.

## How Has This Been Tested?
Tested locally.

URL for testing:
- https://feat--initial-setup--design-website--adobe.hlx3.page/

## Screenshots (if appropriate):
n/a

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by davidnuescheler** [comment]
@pfulton, thanks so much, this is a very impressive first PR ;) you navigated a lot of complexity, especially as `head.html` updates are a bit counter intuitive...

it looks like this PR fails the lighthouse score test, it seems that the impact comes are mostly coming from CLS. 
i think the main contributors are the removal of the CLS protection via the `body` & `body.appear` rules as well as the `data-section-status` CLS protection rules, for block loading and section styling.

on an only vaguely related note, more recently (with a lot of the non webkit rendering engines disappear) we find less value in `normalize.css` (especially as in most projects we don't have explicit support for browsers below 1% traffic).

**Comment by davidnuescheler** [comment]
@rofe i think we should add CSS linting into the boilerplate setup... WDYT?

**Review by davidnuescheler — APPROVED** [review]

File: .stylelintignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+styles/normalize.css
+blocks/carousel/carousel.css
+blocks/example/example.css
+blocks/footer/footer.css
+blocks/header/header.css
+blocks/stories/stories.css
```
Comment (rofe) [inline .stylelintignore:6]: Why are these excluded?

**Review by rofe — CHANGES_REQUESTED** [review]
I'd love to backport `stylelint` to `helix-project-boilerplate`!

File: .stylelintignore (hunk 1 above)
Comment (pfulton) [reply .stylelintignore:6]: Just temporarily ignoring them. I'd like to keep the example code around as a guide, but I also didn't think it was worth my time to fix the linting issues in code that I'm going to (potentially) get rid of.

Happy to fix the issues in each of those files if you'd like me to, though.

File: package.json [hunk 2]
```diff
@@ -35,6 +37,9 @@
     "karma-mocha-reporter": "2.2.5",
     "mocha": "9.1.1",
     "puppeteer": "10.2.0",
-    "shelljs": "0.8.4"
+    "shelljs": "0.8.4",
+    "stylelint": "^14.2.0",
+    "stylelint-config-prettier": "^9.0.3",
+    "stylelint-config-standard": "^24.0.0"
```
[the remark this answers is not in the document]
Comment (pfulton) [reply package.json]: Great suggestion. Thanks!

**Pull request merged**